### PR TITLE
Zoom in from when getting directions

### PR DESCRIPTION
**Pull request by thomasgauvin**

### CODE REVIEW

File: app/src/main/java/com/example/concordia_campus_guide/Fragments/LocationFragment/LocationFragmentViewModel.java [hunk 1]
```diff
@@ -60,6 +60,9 @@
     private BitmapDescriptor currentPOIIcon;
     private Location currentLocation;
 
+    //EV centerCoordinates
+    private LatLng initialZoomLocation = new LatLng(45.495638 ,-73.578258);
```
Comment (viveanban) [inline app/src/main/java/com/example/concordia_campus_guide/Fragments/LocationFragment/LocationFragmentViewModel.java]: Why is this the default? Isn't it supposed to be Hall?

Comment (thomasgauvin) [reply app/src/main/java/com/example/concordia_campus_guide/Fragments/LocationFragment/LocationFragmentViewModel.java]: default was always EV

Comment (viveanban) [reply app/src/main/java/com/example/concordia_campus_guide/Fragments/LocationFragment/LocationFragmentViewModel.java]: mmm Do you think that we should put EV as the "default" currentLocation when we calculate distances b/w POIs too?

**Comment by viveanban** [comment]
Can we also change the floor shown when we do this zoom? Currently if the initial location is a floor lower than the topmost floor, you won't see the directions, until you manually change floors.

**Comment by thomasgauvin** [comment]
@viveanban that's more difficult, need to open a different PR

**Comment by thomasgauvin** [comment]
Tests passing, but system tests need to be fixed (which I have done in my other PR)

File: app/src/main/java/com/example/concordia_campus_guide/Fragments/LocationFragment/LocationFragmentViewModel.java (hunk 1 above)
Comment (tarekait1996) [reply app/src/main/java/com/example/concordia_campus_guide/Fragments/LocationFragment/LocationFragmentViewModel.java:66]: could we add those to class constant just so it is clear which coordinate we are using?

Comment (thomasgauvin) [reply app/src/main/java/com/example/concordia_campus_guide/Fragments/LocationFragment/LocationFragmentViewModel.java:66]: done tarek

Comment (thomasgauvin) [reply app/src/main/java/com/example/concordia_campus_guide/Fragments/LocationFragment/LocationFragmentViewModel.java:66]: and Viveka, no, I don't think so

**Pull request merged**